### DISCUSSION TITLE
Design for calculation steps

### DISCUSSION THREAD

**Discussion by GrimPixel** [bcakmakoglu/vue-flow#733] [Ideas]
The goal is, to write down calculation steps with the flowchart. Each step can be a function in a library, written in Python, C, Fortran, or other languages.

This will bring better clarity to science and engineering, especially for education. I found Vue Flow good to realise it. It would be ideal if even primary school teachers can create them with ease.

**Comment by bcakmakoglu** [comment]
Hey @GrimPixel 
Can you elaborate more what this post is supposed to mean? I'm not sure I understand what the idea behind this actually is 😄

**Comment by GrimPixel** [comment]
Great to hear from you! Here is an example:

Find the minimal coefficient of friction to keep the object on the slope static
https://upload.wikimedia.org/wikipedia/commons/thumb/4/45/Physik-schiefe-Ebene.png/640px-Physik-schiefe-Ebene.png

![Untitled Diagram drawio](https://user-images.githubusercontent.com/39795763/223109286-fb9e037d-5318-49f7-8857-e72e31e2118b.png)
